src/fontra/client/core/var-path.js

```diff
@@ -546,7 +546,7 @@ export class VarPackedPath {
     this._appendPoint(x, y, VarPackedPath.ON_CURVE);
   }
 
-  cubicCurveTo(x1, y1, x2, y2, x3, y3) {
+  bezierCurveTo(x1, y1, x2, y2, x3, y3) {
```

Comment:
This change is what is causing the js unit tests fail. Instead, please add a *new* method `bezierCurveTo`, that simply calls `cubicCurveTo`, and document it as a "compatibility method for Path2D".

Reply:
Thanks. I actually missed to mention that I was aware of the failing unit test, but wanted to talk to you about how we should deal with that. (Not enough time yesterday.) If I should adjust the unittest or a different solution. Now I know how to proceed :) 
Will fix it immediately.

Reply:
I don't think this change is needed anymore.

Reply:
Do you want me to remove it, or do we keep it to be in sync with 'Path2D'?

Reply:
It does not need to be in sync. It is not needed, so should be removed.

Reply:
OK. Done.